docs/source/_json/batching.json

```diff
@@ -0,0 +1,48 @@
+GET /plone/folder/@search?b_size=5&sort_on=path
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+{
+  "@id": "http://localhost:55001/plone/folder/@search?sort_on=path", 
```

Comment:
@lukasgraf if the request is on "/folder/@search?b_size=5&sort_on=path", why is the @id equals "/folder/@search?sort_on=path" (without the "b_size")? Shouldn't this be either the base request or the full request with all params?